readthedocs/proxito/views/hosting.py

```diff
@@ -222,54 +218,6 @@ def get(self, request, *args, **kwargs):
         return JsonResponse(data, json_dumps_params={"indent": 4, "sort_keys": True})
 
 
-class NoLinksMixin:
-
-    """Mixin to remove conflicting fields from serializers."""
-
-    FIELDS_TO_REMOVE = ("_links",)
```

Comment:
Adding links back may cause a snowball of dependencies, requiring to have all endpoints exposed in the proxied API. I wonder if another alternative can be to have a different and more restricted API under the addons umbrella instead of trying to move the whole API.

Reply:
I want to avoid having two different API responses one from the dashboard and one from proxito. That will increase maintenance work and also will add confusion. Let's try how it works with this initial implementation that just proxies the APIv3 but read-only using the exact same response for now and see how it goes.